django/test/runner.py

```diff
@@ -424,6 +426,51 @@ def __iter__(self):
         return iter(self.subsuites)
 
 
+class TimingTextTestResult(unittest.TextTestResult):
+    def __init__(self, time_keeper, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.start_time = None
+        self.time_keeper = time_keeper
+
+    def print_time(self, test):
+        total = time.perf_counter() - self.start_time
+        self.start_time = None
+        if self.showAll:
+            self.stream.write("{:.3f}s ".format(total))
```

Comment:
```suggestion
            self.stream.write('%.3fs ' % total))
```
Do we need a trailing space?

Reply:
No, we shouldn't need it. The stream gets flushed after every test